addons/mail/static/src/core/common/composer.js

```diff
@@ -87,6 +87,7 @@ export class Composer extends Component {
     setup() {
         super.setup();
         this.isMobileOS = isMobileOS();
+        this.isIOS = isIOS();
```

Comment:
Since you mentioned it, I’ve added `isIosApp()`. However, I believe we still need `isIOS()` for scenarios where the user opens the discussion in a browser on iOS devices.

Reply:
The bottom margin is a non-issue in browser, because there's a navigation menu at bottom of screen which doesn't overlap with the swipe gesture.

So this is an issue specific to PWA on iOS.